ee/ucp/admin/configure/restrict-services-to-worker-nodes.md

```diff
@@ -81,29 +71,38 @@ $ kubectl get <object> <workload> -o json | jq -r '.spec.template.spec.toleratio
 }
 ```
 
-### Allow Users and Service Accounts to Schedule on Manager / DTR Nodes
+### Allow users and service accounts to schedule workloads on manager / DTR nodes
 
-To allow Kubernetes Users and Service Accounts to deploy workloads accross all
-node types in your cluster, an Administrator will need to tick "Allow all
-authenticated users, including service accounts, to schedule on all nodes,
-including UCP managers and DTR nodes." in the UCP web interface. 
+To allow Kubernetes users and service accounts to deploy workloads across all node types in a cluster, an administrator must select the **Allow all authenticated users, including service accounts, to schedule on all nodes, including UCP managers and DTR nodes.** check box in the UCP UI. 
 
 ![](../../images/restrict-services-to-worker-nodes-3.png){: .with-border}
 
-For all new workloads deployed by Kubernetes Users after this box has been
-ticked, UCP will apply a toleration to your workloads to allow the pods to be
-scheduled on all node types. For existing workloads, the User would need to edit
-Pod Specification as detailed above in the "Allow Administrators to Schedule on
-Manager / DTR Nodes" section.
+When this option is enabled, for all new workloads deployed by Kubernetes users, UCP will apply a toleration to workloads to allow the pods to be scheduled on all node types. 
```

Comment:
Suggest removing the first comma (after 'enabled').

Also, suggest changing "to allow" to "that allows".

Reply:
See above comment.

Reply:
See above comment to comment. :-)